etc/service/postgresql/run

```diff
@@ -30,7 +30,7 @@ if ! [[ $(ls -A "${DATADIR}" | grep -v placeholder) ]]; then
     pgsql_command "CREATE ROLE ${PGSQL_ROLE_1_USERNAME} WITH ${PGSQL_ROLE_1_FLAGS} ENCRYPTED PASSWORD '${PGSQL_ROLE_1_PASSWORD}';"
   fi
   if [[ -n "${PGSQL_DB_1_NAME}" && -n "${PGSQL_DB_1_OWNER}" ]]; then
-    pgsql_command "CREATE DATABASE ${PGSQL_DB_1_NAME} WITH OWNER ${PGSQL_DB_1_OWNER} ENCODING '${PGSQL_DB_1_ENCODING:-UNICODE}';"
+    pgsql_command "CREATE DATABASE ${PGSQL_DB_1_NAME} WITH OWNER ${PGSQL_DB_1_OWNER} ENCODING '${PGSQL_DB_1_ENCODING:-UNICODE}' LC_COLLATE '${PSQL_DB_1_LC_COLLATE}' LC_CTYPE '${PSQL_DB_1_LC_CTYPE}' TEMPLATE ${PSQL_DB_1_TEMPLATE:-DEFAULT};"
```

Comment:
I thought the same, but the examples in these two documents shows unquoted:

- https://www.postgresql.org/docs/9.3/static/sql-createdatabase.html
- https://confluence.atlassian.com/adminjiraserver071/connecting-jira-applications-to-postgresql-802592176.html